js/src/main/scala/com/raquo/domtypes/jsdom/defs/events/ElementTargetEventWithValue.scala

```diff
@@ -0,0 +1,16 @@
+package com.raquo.domtypes.jsdom.defs.events
+
+import org.scalajs.dom
+
+/** One of the possible ways to access `target.value` on input events.
+  * If you want to use this, define an implicit conversion that instantiates this class.
```

Comment:
I mean, the only benefit of keeping that class implicit would have been to let end users import the implicit class from SDT. But I doubt that the consuming library would want end users to be doing that, said library will probably have its own way of managing the implicits it needs.

Reply:
Good point, I agree.

Reply:
Hmm, while using it i feel, this extension might not belong into this library. In the end, I might be going to extend it in the consuming library with things like: `valueAsNumber` or `checked` or go for another concept with something like `def typedTarget[T <: dom.Element] = target.asInstanceOf[T]`. For transparency to other developers it then makes sense to define them in one place where it is visible.